src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1684,6 +1688,34 @@ TypeSymbol getRightResultType(TypeSymbol leftType, TypeSymbol rightType)
             }
         }
 
+        private static bool? GetIsNullable(BoundExpression expr)
```

Comment:
>GetIsNullable [](start = 29, length = 13)

xml doc would be useful.
The method name and the behavior didn't quite chime for me. In particular, we're returning `null` in many more cases than what "IsNullable" would denote.